content/applications/inventory_and_mrp/inventory/product_management/product_tracking/package.rst

```diff
@@ -48,147 +44,85 @@ the :guilabel:`Operations` heading, activate the :guilabel:`Packages` feature. T
    :align: center
    :alt: Activate the *Packages* setting in Inventory > Configuration > Settings.
 
-.. _inventory/warehouses_storage/package-use:
-
-Package use
-===========
-
-.. note::
-   The following section is **only** necessary when packages are intended to be used for cluster
-   pickings.
-
-.. seealso::
-   :doc:`Cluster picking <../../warehouses_storage/advanced_operations_warehouse/cluster_picking>`
-
-First, navigate to :menuselection:`Inventory app --> Configuration --> Settings`, and activate the
-:guilabel:`Batch Transfers` feature, located in the :guilabel:`Operations` section. Doing so makes
-the *Package Use* field appear on a package form.
+.. _inventory/warehouses_storage/pack:
 
-.. image:: package/enable-batch.png
-   :align: center
-   :alt: Activate the *Batch Transfers* feature in Inventory > Configuration > Settings.
+Pack items
+==========
 
-Then, go to the package form to configure the package intended for batch picking. To do that,
-navigate to :menuselection:`Inventory app --> Products --> Packages`.
+Products can be added to packages in any transfer by:
 
-On this page, add new packages by clicking :guilabel:`New`, or select an existing package. Doing so
-opens the package form, which contains the following fields:
+#. Clicking each *Detailed Operations* icon on the product line.
+#. Using the *Put in Pack* button to place everything in the transfer into a package.
```

Comment:
would you think it would be beneficial to add some anchored links here, to better signify that these elements are about to be explained? That could be useful